src/go/k8s/pkg/resources/certmanager/certificate.go

```diff
@@ -42,6 +42,7 @@ type CertificateResource struct {
 	issuerRef    *cmetav1.ObjectReference
 	fqdn         string
 	isCA         bool
+	wildcard     bool
```

Comment:
nit: haha I see what you did there :) I think nicer would be to have two constructors, one will accept common name, one fqdn and all this wildcard and everything will be hidden underneath. But I am fine with making that as a follow up :)

Reply:
I've addressed this in another commit to this PR, let me know if it's 👍 @dimitriscruz